service/history/tasks/category.go

```diff
@@ -41,12 +39,12 @@ type (
 )
 
 const (
```

Comment:
nit: would you mind help add a comment that says those IDs are persisted in DB and can't not be changed once defined?

Reply:
Done